infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -37,14 +37,14 @@ public LogReplicationMetadataManager(CorfuRuntime rt, long topologyConfigId, Str
         this.corfuStore = new CorfuStore(runtime);
         metadataTableName = getPersistedWriterMetadataTableName(localClusterId);
         try {
-            metadataTable = this.corfuStore.openTable(namespace,
+            metadataTable = this.corfuStore.openTable(NAMESPACE,
                             metadataTableName,
                             LogReplicationMetadataKey.class,
                             LogReplicationMetadataVal.class,
                             null,
                             TableOptions.builder().build());
         } catch (Exception e) {
-            log.error("Caught an exception while opening the table namespace={}, name={}", namespace, metadataTableName);
+            log.error("Caught an exception while opening the table NAMESPACE={}, name={}", NAMESPACE, metadataTableName, e);
```

Comment:
looks like the exception wont get printed?

Reply:
Yes it will, for exceptions we don't need to specify a place holder in log.error